dist/index.js

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
I wouldn't commit the dist folder to avoid merge conflicts when developing for meta-diagram. I understand that this would make it easy to directly use this package from github in other projects but with that in consideration I would still not keep it.